.github/workflows/create_tagged_releases.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         shell: bash
         run: |
           echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
-          echo "branch=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_OUTPUT
```

Comment:
`GITHUB_REF#refs/heads/` contains `ref/tag` which we don't want if we are going to use it as version tag, not sure if this was the intended way for this var or another one should be used as the input was missing for the action